src/test/java/org/inferred/freebuilder/processor/AnalyserTest.java

```diff
@@ -406,6 +406,81 @@ public void abstractButNotGetter() throws CannotGenerateCodeException {
   }
 
   @Test
+  public void abstractMethodNamedGet() throws CannotGenerateCodeException {
+    Metadata dataType = analyser.analyse(model.newType(
+        "package com.example;",
+        "public class DataType {",
+        "  public abstract String get();",
+        "  public static class Builder extends DataType_Builder {}",
+        "}"));
+    assertThat(dataType.getProperties()).isEmpty();
+    assertThat(messager.getMessagesByElement().keys()).containsExactly("get");
+    assertThat(messager.getMessagesByElement().get("get"))
+        .containsExactly("[ERROR] Only getter methods (starting with 'get' or 'is') may be declared"
+            + " abstract on @FreeBuilder types");
+  }
+
+  @Test
+  public void abstractMethodNamedGetter() throws CannotGenerateCodeException {
+    Metadata dataType = analyser.analyse(model.newType(
+        "package com.example;",
+        "public class DataType {",
+        "  public abstract String getter();",
+        "  public static class Builder extends DataType_Builder {}",
+        "}"));
+    assertThat(dataType.getProperties()).isEmpty();
+    assertThat(messager.getMessagesByElement().keys()).containsExactly("getter");
+    assertThat(messager.getMessagesByElement().get("getter"))
+        .containsExactly("[ERROR] Getter methods cannot have a lowercase character immediately"
+            + " after the 'get' prefix on @FreeBuilder types (did you mean 'getTer'?)");
+  }
+
+  @Test
+  public void abstractMethodNamedIssue() throws CannotGenerateCodeException {
+    Metadata dataType = analyser.analyse(model.newType(
+        "package com.example;",
+        "public class DataType {",
+        "  public abstract String issue();",
+        "  public static class Builder extends DataType_Builder {}",
+        "}"));
+    assertThat(dataType.getProperties()).isEmpty();
+    assertThat(messager.getMessagesByElement().keys()).containsExactly("issue");
+    assertThat(messager.getMessagesByElement().get("issue"))
+        .containsExactly("[ERROR] Getter methods cannot have a lowercase character immediately"
+            + " after the 'is' prefix on @FreeBuilder types (did you mean 'isSue'?)");
+  }
+
+  @Test
+  public void abstractMethodNamedGetürkt() throws CannotGenerateCodeException {
+    Metadata dataType = analyser.analyse(model.newType(
+        "package com.example;",
+        "public class DataType {",
+        "  public abstract String getürkt();",
+        "  public static class Builder extends DataType_Builder {}",
+        "}"));
+    assertThat(dataType.getProperties()).isEmpty();
+    assertThat(messager.getMessagesByElement().keys()).containsExactly("getürkt");
+    assertThat(messager.getMessagesByElement().get("getürkt"))
+        .containsExactly("[ERROR] Getter methods cannot have a lowercase character immediately"
+            + " after the 'get' prefix on @FreeBuilder types (did you mean 'getÜrkt'?)");
```

Comment:
You may also want to add a test that does:
Locale.setDefault(Locale.forLanguageTag("tr-TR"))
Then make sure getint is converted to getInt rather than getİnt